### DISCUSSION TITLE
Timeout viewing ticket

### DISCUSSION THREAD

**Discussion by svp95bos** [osTicket/osTicket#6416] [Q&A] [answered]
Hi,

We're getting a timeout exception when trying to view a ticket.

![image](https://user-images.githubusercontent.com/36014542/208656517-0cd24883-965c-4b49-816e-f02dce9fe52b.png)

Thing is, we've recently applied our own translation file instead of the default `sv_SE.phar`.

Our `i18n\sv_SE` folder looks like this:

![Skärmbild 2022-12-20 121009](https://user-images.githubusercontent.com/36014542/208653337-b11d5ac7-015e-40ef-a78d-01ac4db65da5.png)

where these folders contain the same files as in the extracted phar file, except for `LC_MESSAGES\messages.mo.php` which has been updated.

This is the `messages.mo.php` file: 

[messages.mo.php.txt](https://github.com/osTicket/osTicket/files/10267908/messages.mo.php.txt)

Error logged (I don't know if this is relevant for this issue):

`[20-Dec-2022 11:32:26 UTC] PHP Fatal error:  Maximum execution time of 300 seconds exceeded in D:\osticket\include\class.schedule.php on line 855`

Before we replaced the `sv_SE.phar` file w/ the above, everything worked as excpected. I can also add that everything works as before, except for when trying to view some of the tickets (other tickets work just fine). I wonder if it has something to do w/ the contents of some of our tickets that are causing this error.

![image](https://user-images.githubusercontent.com/36014542/208656766-335e22a0-cf9a-4e0b-8846-2abdfdb91cbc.png)

Kind regards,
Peter

**Comment by JediKev** [comment, marked as answer]
@svp95bos 

Please use [our Forum](https://forum.osticket.com) for things like this.

Cheers.
